src/scripts/export-page.js

```diff
@@ -163,7 +164,10 @@ H5P.DocumentExportPage.ExportPage = (function ($, EventDispatcher) {
   ExportPage.prototype.initExportButton = function () {
     var self = this;
     // Export document button event
-
+    // Quick and Dirty solution for disabling document exports on mobile devices
+    if (isMobile) {

```

Comment:
Scanning react-device-detect, looks like this isn't going to be set for tablets (or embedded/wearables either)